src/server/verification/verificationAPI.js

```diff
@@ -182,6 +184,29 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'verificationAPI - verify/topwallet' })
       const user: LoggedUser = req.user
+      // check if user send ether out of the good dollar system
+      let isUserSendEtherOutOfSystem = false
+      try {
+        const { result = [] } = await fuseapi.getTxList({
+          address: user.gdAddress,
+          page: 1,
+          offset: 0,
+          filterby: 'from'
+        })
+        isUserSendEtherOutOfSystem = result.some(r => Number(r.value) > 0)
```

Comment:
I feel this is too strict
maybe check if there are >1 such transactions so it looks like he is doing it repeatedly in a short time period

Reply:
but I think user mustn't use ether at all out of system.